design/FY2021/NFP-Hour_Zero_Treatment.md

```diff
@@ -0,0 +1,85 @@
+Adding input option for Hour Zero weather data treatment
+================
+
+**J. Yuan & M.J. Witte, GARD Analytics**
+
+ - Original Date - 2/8/21
+ - Revision Date
+
+
+## Justification for New Feature ##
+
+This topic deals with how the first few sub-hourly weather data should be used. Currently in the code, Hr 1 and Hr 24 data was used as the base for the interpolation for the first hour time steps. However, intrinsically, there is no phsyical connection between Hr 1 and Hr 24 weather data, which can be observed in many other weather files (e.g. Chicago ORD airport 725300, a similar large Hr 1 and Hr 24 difference can be observed). To a user’s point view, this is might also a little bit difficult to interprete the first few time step output weather data , without knowing the reported values actually show the interpolated results with the data from Hour 1 and Hour 24.
+
+There were also a little bit of history that this setting had changed back and forth a few time in the past. Here we proposed to add options to allow both these two options. They shoudl differ minimally for most simulations. However, it would allow the user to choose to use a "natural" way of using Hr 1 data for Hr 0 in the interpolation. 
+
+## E-mail and  Conference Call Conclusions ##
+
+
+
+## Overview ##
+
+The proposal seeks to add an input to allow for the choice of using Hr 1 or Hr 24 for the first sub-hour timesteps' interpolation of weather data. 
+
+## Approach ##
+
+After considering a few places such as "Simulation Control", "TimeStep", and "Run Period", it is proposed to the additional input field at the end of "Run Period."
+
+Original:
+
+RunPeriod,
+       \memo Specify a range of dates and other parameters for a simulation.
+       \memo Multiple run periods may be input, but they may not overlap.
+	...	   
+  A8;  \field Treat Weather as Actual
+       \type choice
+       \key Yes
+       \key No
+       \default No
+	  
+Proposed: 
+
+RunPeriod,
+       \memo Specify a range of dates and other parameters for a simulation.
+       \memo Multiple run periods may be input, but they may not overlap.
+	...	   
+  A8,  \field Treat Weather as Actual
+       \type choice
+       \key Yes
+       \key No
+       \default No
+  A9, First Hour Interpolation Starting Values
+       \note When the weather data timestep is longer than the simulation timestep, weather data is interpolated. For the first hour of the
+       \note simulation, this field specifies which values from the first day of the run period to use as the interpolation starting point.
+       \note This same interpolation will be used for repeated warmup days.
+	   \key Hour1
+       \key Hour24
+       \default Hour24
```

Comment:
You could also add a field to allow the user to set the previous hour temperature to use for the warmup and initial time step of the run period.

Reply:
@rraustad Thanks. This is a good suggestion. With this new  option it could even allow the user the choose where to use which option---I was thinking it could be use Hr24 for the warmup and Hr1 for initial time step of a long run period by default.